main.js

```diff
@@ -36,6 +39,7 @@ const initKmtHeader = () => {
               last: true
             },
             {
+              tracking: "my-account",
```

Comment:
Think this could be done with attrs: { 
 data-trackable: "whatever"
} 

because the {...item.attrs} will pck that up in hte link tag on line 27.
Not sure which is the best way though